feature-libs/asm/package.json

```diff
@@ -34,9 +34,9 @@
     "@ngrx/store": "^14.3.0",
     "@spartacus/cart": "4.1.0-next.0",
     "@spartacus/core": "4.1.0-next.0",
-    "@spartacus/schematics": "4.1.0-next.0",
```

Comment:
I have noticed a lot of feature libs version changes here, that are not present in develop. Are those necessary for s4om?